tests/core/plan-env-file/data/plan.fmf

```diff
@@ -0,0 +1,40 @@
+/test:
+    test: echo "MYVAR1=\"${MYVAR1}\" MYVAR2=\"${MYVAR2}\""
+
+/plan:
+    discover:
+        how: fmf
+    provision:
+        how: local
+    execute:
+        how: tmt
+    finish:
+        how: shell
+        script: echo "FINISH1=\"${MYVAR1}\" FINISH2=\"${MYVAR2}\""
+
+    /single:
+        prepare:
+            how: shell
+            script: echo "MYVAR1=MYVAR1_VALUE" >> $TMT_PLAN_ENVIRONMENT_FILE
+
+    /multiple:
+        prepare:
+            how: shell
+            script: ./setup_env.sh
+
+    /env-file-override:
+        prepare:
+            how: shell
+            script: echo "MYVAR1=MYVAR1_VALUE" >> $TMT_PLAN_ENVIRONMENT_FILE
+        environment-file:
+            - override.env
+
+    /execute-override:
+        discover:
+            how: shell
```

Comment:
The `discover` step here is not needed as `script` is used under `execute`. Or we can move the test script here.

Reply:
Fixed in 9ecf336.